x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/details_page/details_page_mappings_content.tsx

```diff
@@ -414,12 +415,15 @@ export const DetailsPageMappingsContent: FunctionComponent<{
       >
         {showAboutMappings && (
           <EuiFlexItem
```

Comment:
why not creating a constant with the styles at the top of the component like this:
const showAboutMappingsStyles = css({
    min-width: 400px,
    [useEuiBreakpoint(['xl'])]: {
      max-width: 480px
    },
  });
and then just added to the item?
```suggestion
          <EuiFlexItem
          grow={false}
          css={showAboutMappingsStyles}>
```
This way we'll avoid calling a hook conditionally

Reply:
Makes sense! Thanks.  Fixed in my latest commit